src/domain/executable.ts

```diff
@@ -161,7 +161,7 @@ export abstract class Executable {
       ref,
     }))
 
-    return yield* this.domainManager.addSteps(parsedDomains, false)
+    return yield* this.domainManager.addSteps(parsedDomains, 'override')
```

Comment:
shouldn't it be 'ignore' here to keep the old behaviour when we are not trying to update them?

Reply:
Fixed